cmake/CMakeLists.txt

```diff
@@ -56,7 +56,6 @@ option(onnxruntime_ENABLE_PYTHON "Enable python buildings" OFF)
 option(onnxruntime_ENABLE_MEMLEAK_CHECKER "Experimental: Enable memory leak checker in Windows debug build" OFF)
 option(onnxruntime_USE_CUDA "Build with CUDA support" OFF)
 option(onnxruntime_USE_OPENVINO "Build with OpenVINO support" OFF)
-option(onnxruntime_USE_NSYNC "Build with NSYNC support. This option only takes effect on Linux" OFF)
```

Comment:
Is nsync going to be ON by default? If yes, do we still need ```#ifdef USE_NSYNC```?

Reply:
NO. We can remove the ifdef

Reply:
PR sent: #3052